apps/fluent-tester/src/E2E/BootPage/specs/BootPages.spec.win.ts

```diff
@@ -118,6 +119,12 @@ describe('Click on each test page and check if it renders', function () {
     expect(TextTestPage.isPageLoaded()).toBeTruthy();
   });
 
+  it('Tabs Test Page', () => {
```

Comment:
Does this also work on UWP? Should it be added to the Windows E2E testing as well?

Reply:
It does not work because there is no FocusZone on UWP. The Tabs UWP Control is still a work in progress. We will probably submit another PR for it.